tests/system/small/test_session.py

```diff
@@ -405,6 +405,24 @@ def test_read_gbq_with_configuration(
     assert df.shape == (9, 3)
 
 
+def test_read_gbq_with_custom_global_labels(
+    session: bigframes.Session, scalars_table_id: str
+):
+    bigframes.options.compute.assign_extra_query_labels(test1=1, test2="abc")
+    bigframes.options.compute.extra_query_labels["test3"] = False
+
+    job_labels = session.read_gbq(scalars_table_id).query_job.labels  # type:ignore
```

Comment:
wonder what type error you get here?

Reply:
the query job can return None type, which don't have .labels